packages/@expo/cli/src/utils/exit.ts

```diff
@@ -131,6 +141,11 @@ export function ensureProcessExitsAfterDelay(waitUntilExitMs = 10000, startedAtM
     // Check if the process can exit
     ensureProcessExitsAfterDelay(waitUntilExitMs, startedAtMs);
   }, 100);
+
+  // Unref the timeout so it doesn't prevent the process from exiting naturally
+  // when this timeout is the only remaining active resource
+  // @ts-expect-error: unref is a method on Timeout in Node.js
```

Comment:
We should avoid `ts-expect-error` as much as possible (as per the recent PR that removes all leftovers), since it's the "loosest" of all casts. Since it's just a comment it doesn't prevent the underlying type error and instead filters it out of the output. This also means that it makes us blind to any error when it potentially shifts or other (unintended) errors occur.

In this case, we can likely cast to Node's timeout

re. while it makes sense to keep for now if the change in `delay.ts` is insufficient, we should probably leave a TODO, since if this is still needed this indicates we have a timeout leak somewhere else

Reply:
Good feedback, seems like the react-native global types were being used. Tried changing them but it was a mess so using type casting for now.